README.md

```diff
@@ -161,7 +163,39 @@ Bind<IStatsDPublisher>().To<StatsDPublisher>().InSingletonScope();
 
 The default behaviour is to ignore the error.
 
-#### Example of using the interface
+#### Tagging support
+
+Tags or dimensions are not covered by the StatsD specification. Providers supporting tags have implemented their flavours. Some of the major providers are supported out-of-the-box:
+
+* [CloudWatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-custom-metrics-statsd.html).
+* [DataDog](https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics)
+* [InfluxDB](https://www.influxdata.com/blog/getting-started-with-sending-statsd-metrics-to-telegraf-influxdb/#introducing-influx-statsd).
+* [Librato](https://github.com/librato/statsd-librato-backend#tags).
+* [SignalFX](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-statsd.html#adding-dimensions-to-statsd-metrics).
+* [Splunk](https://docs.splunk.com/Documentation/Splunk/8.0.5/Metrics/GetMetricsInStatsd).
+
+```csharp
+var config = new StatsDConfiguration
+{
+    Prefix = "prefix",
+    Host = "127.0.0.1",
+    TagsFormatter = TagsFormatter.CloudWatch,
+};
+```
+
+##### Extending tags formatter
+
+As tags are not part of the StatsD specification, the `IStatsDTagsFormatter` used can be extended and injected in the `StatsDConfiguration`.
+
+The template class `StatsDTagsFormatter` can be inherited providing the `StatsDTagsFormatterConfiguration`:
+
+* **Prefix**: the string that will appear before the tag(s).
+* **Suffix**: the string that will appear after the tag(s).
+* **AreTrailing**: a boolean indicating if the tag(s) are placed at the end of the StatsD message (like it is supported by AWS CloudWatch, DataDog or Splunk) or otherwise they are right after the bucket name (like it is supported by InfluxDB, Librato or SignalFX).
```

Comment:
```suggestion
* **AreTrailing**: a boolean indicating if the tag(s) are placed at the end of the StatsD message (like it is supported by AWS CloudWatch, DataDog and Splunk) or otherwise they are right after the bucket name (like it is supported by InfluxDB, Librato and SignalFX).
```